addons/pause/userscript.js

```diff
@@ -16,6 +16,16 @@ export default async function ({ addon, global, console, msg }) {
   onPauseChanged(setSrc);
 
   while (true) {
+    if (!document.querySelector(".pause-btn")) {
+      document.addEventListener("keydown", function (e) {
+        if (addon.settings.get("keybindEnabled")) {
+          if (e.altKey && e.key === "≈") {
+            e.preventDefault();
+            img.click();
```

Comment:
```suggestion
            setPaused(!isPaused())
```

Reply:
> 

Would that allow the pause button image to change as well?

Reply:
It should, I believe.